lm75bd/lm75bd.c

```diff
@@ -28,7 +28,43 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  error_code_t errCodeSend;
+  error_code_t errCodeReceive;
```

Comment:
you can just use one `errCode` instead of using a separate one for each function